### ISSUE TITLE
nologin shell missing on vestacp

### ISSUE THREAD

**Issue by pablobae** [serghey-rodin/vesta#1792]
### Operating System (OS/VERSION):

Centos 7

### VestaCP Version:

Latest .23

### Installed Software (what you got with the installer):

Nginx+apache + ftp + dns + mail + spam...

### Steps to Reproduce:

Install Centos 7
Install Vesta
Login into vestapanel
Add user, and try to assign "nologin" shell.

### Other Notes:
Nologin opcion is missing.

**Comment by alex-connor** [comment]
Check the contents of the file / etc / shells, should be
> /bin/sh
> /bin/bash
> /sbin/nologin
> /usr/bin/sh
> /usr/bin/bash
> /usr/sbin/nologin

If the content of the file does not match, then you need to fix it.

**Comment by pablobae** [comment]
Thank you! Is it a issue of vesta or Centos?

**Comment by alex-connor** [comment]
CentOS, and maybe other OS

**Comment by Skamasle** [comment]
I can cofirm than centos have this problem but is not present in debian 9

In debian 9 for any reason nologin shell is duplicated, maybe add it twice

```
# /etc/shells: valid login shells
/bin/sh
/bin/dash
/bin/bash
/bin/rbash
/sbin/nologin
/usr/sbin/nologin
/usr/bin/rssh
/sbin/nologin
```

**Comment by anton-reutov** [comment]
Если возможно, то проверьте на самой последней версии CentOS, не повторяется.

**Comment by anton-reutov** [comment]
Try to check this bug on the last version of CentOS 7.6, can't reproduce

**Comment by pablobae** [comment]
It's fixed on 7.6

> 
> 
> Try to check this bug on the last version of CentOS 7.6, can't reproduce
